mixin/dashboards/compact.libsonnet

```diff
@@ -18,7 +19,7 @@ local g = import '../lib/thanos-grafana-builder/builder.libsonnet';
             'Shows rate of execution for compactions against blocks that are stored in the bucket by compaction group.'
           ) +
           g.queryPanel(
-            'sum(rate(thanos_compact_group_compactions_total{namespace="$namespace",job=~"$job"}[$interval])) by (job, group)',
+            'sum(rate(thanos_compact_group_compactions_total{%(namespaceLabel)s="$namespace",%(selector)s}[$interval])) by (job, group)' % thanos.compact,
```

Comment:
I'm not sure about job to selector change. It's specified as `job` here because we also aggregate on job for these queries and we have a dropdown for this variable. Rather than removing job we should have additional `selector` if it's needed.

Reply:
Makes sense, I can selector as an additional one.